projects/1.18/assets/xaeros-world-map/xaeroworldmap/lang/zh_cn.json

```
@@ -190,5 +190,33 @@
   "gui.xaero_wm_reset_message2": "此操作无法撤消！",
   "gui.xaero_wm_partial_y_teleportation": "传送时使用偏移的Y坐标",
   "gui.xaero_wm_box_partial_y_teleportation": "当传送到地图位置时，将Y坐标偏移0.5，以避免穿过地毯状的方块落入虚空。否则，将使用整数Y坐标。",
-  "gui.xaero_wm_display_stained_glass": "显示染色玻璃"
-}
\ No newline at end of file
+  "gui.xaero_wm_display_stained_glass": "显示染色玻璃",
+  "gui_xaero_wm_dropdown_map_select" : "地图选择",
+  "gui.xaero_wm_hovered_biome" : "鼠标处生物群系",
+  "gui.xaero_wm_unknown_biome" : "未知的生物群系",
+  "gui.xaero_wm_cave_mode_depth" : "洞穴模式深度",
+  "gui.xaero_box_update_chunks" : "Update chunks on the non-world-save maps (multiplayer).",
+  "gui.xaero_box_load_chunks" : "Add new chunks to the non-world-save maps (multiplayer).",
+  "gui.xaero_wm_legible_cave_maps" : "清晰可辨的洞穴地图",
+  "gui.xaero_wm_box_legible_cave_maps" : "在洞穴模式中使用基于深度的照明而不是实际方块照明，从而更容易在心理上区分不同的洞穴层。",
+  "gui.xaero_box_cave_mode" : "洞穴模式",
+  "gui.xaero_box_cave_mode_not_allowed" : "洞穴模式（未被允许）",
+  "gui.xaero_wm_cave_mode_start_auto" : "自动",
+  "gui.xaero_wm_cave_mode_start" : "洞穴模式顶部Y坐标",
+  "gui.xaero_wm_cave_mode_type" : "洞穴模式类型",
+  "gui.xaero_wm_box_cave_mode_type" : "启用洞穴模式后使用的洞穴模式类型。 \n \n 关 - 即使自动/手动启用了洞穴模式，仍显示地面上的地形（与没有洞穴模式的旧版世界地图相同） \n 分层 - 根据目前使用的顶部Y坐标将洞穴地图分成若干层 \n 整体 - 无视目前使用的顶部Y坐标，使用单层洞穴地图，测绘从顶部到底部的所有地形",
+  "gui.xaero_wm_default_cave_mode_type" : "洞穴模式默认类型",
+  "gui.xaero_wm_box_default_cave_mode_type" : "指定世界/服务器中首次访问的维度的洞穴模式默认类型。 你可以单击地图界面上的洞穴模式图标，阅读洞穴模式类型的详细信息。",
+  "gui.xaero_wm_cave_mode_type_layered" : "分层",
+  "gui.xaero_wm_cave_mode_type_full" : "整体",
+  "gui.xaero_auto_cave_mode" : "自动洞穴模式",
+  "gui.xaero_box_auto_cave_mode" : "查看全屏地图时，模组应如何确定“自动”顶部Y坐标。 天花板大小是指固体方块构成的正方形天花板的大小，洞穴模式必须要在你上方检测到它才能激活。 固体方块不必位于同一Y坐标的水平面上。\n \n 小地图 - 若安装了小地图模组，会自动匹配它的洞穴模式，否则使用 3x3 的天花板大小",

```

同上，可能不需要的空格，需要确认没有换行需求
```suggestion
  "gui.xaero_box_auto_cave_mode" : "查看全屏地图时，模组应如何确定“自动”顶部Y坐标。天花板大小是指固体方块构成的正方形天花板的大小，洞穴模式必须要在你上方检测到它才能激活。固体方块不必位于同一Y坐标的水平面上。\n \n 小地图 - 若安装了小地图模组，会自动匹配它的洞穴模式，否则使用3x3的天花板大小",
```